docs/envvars.rst

```diff
@@ -97,6 +97,9 @@ See :ref:`automatic-pypi` for more about PyPI uploads.
 -----------------
 Used together with :ref:`env-pypi_username` when publishing to https://pypi.org/.
 
+.. note::
+  If :ref:`env-pypi_password`, :ref:`env-pypi_username`, and :ref:`env-pypi_token` are not specified credentials from ``$HOME/.pypirc`` will be used.
+
```

Comment:
I would place this under the `PYPI_TOKEN` heading.

Reply:
Thanks! I wasn't sure where to put it.